meshroom/core/node.py

```diff
@@ -458,7 +458,7 @@ def _buildCmdVars(self):
             self._cmdVars[name] = '--{name} {value}'.format(name=name, value=v)
             self._cmdVars[name + 'Value'] = str(v)
 
-            if v is not None and v is not '':
+            if v not in (None, ''):
```

Comment:
Wouldn't it be better to just do: `if v:`?

Reply:
Indeed, `v` is a string value at this point. Only testing `if v:` is more relevant here.
By the way, it would be also nice to add Python 3.7 variant in appveyor.yml and .travis.yml.